src/routes/namespaces/[namespace]/workflows/[workflow]/[run]/_header.svelte

```diff
@@ -8,32 +11,30 @@
   export let namespace: string;
 </script>
 
-<header class="flex flex-col justify-between">
-  <main class="px-6 mb-2">
-    <div class="flex m-0 mt-6 justify-between items-center">
-      <h1 class="text-lg">
-        <a href={`/namespaces/${namespace}/workflows`}>
-          <i
-            class="fas fa-arrow-left inline w-5 h-5 text-gray-500 cursor-pointer"
-          />
-        </a>
+<header class="flex flex-col gap-4">
+  <main class="flex flex-col gap-1 relative">
+    <a
+      href="/namespaces/{namespace}/workflows"
+      class="absolute top-2 back-to-workflows"
+      style="left: -1.5rem"
```

Comment:
When we upgrade to Tailwind 3, we can use the `-left-6` class to get the same effect.